Sources/SwiftProtobuf/ProtobufBinaryTypes.swift

```diff
@@ -606,6 +666,12 @@ public extension ProtobufString {
         }
         throw ProtobufDecodingError.invalidUTF8
     }
+
+    public static func encodedSize(of value: String) -> Int {
+        let stringWithNul = value.utf8CString
```

Comment:
Want dump this idea into an issue so we don't drop it?  (I wasn't sure the if `String.UTF8View` would be cheaper since this seems to make a cstring just to get the length.